task_7/andrew.samsonov/build.gradle

```diff
@@ -0,0 +1,14 @@
+group 'ua.pp.darknsoft'
+version '1.0-SNAPSHOT'
+
+apply plugin: 'java'
+
+sourceCompatibility = 1.5
```

Comment:
Your sources are compiled as if you have only Java 5 version.
Use 1.8 for example.